backend/test/deal-observer.test.js

```diff
@@ -100,3 +103,59 @@ describe('deal-observer-backend', () => {
     assert.strictEqual(await countStoredActiveDeals(pgPool), 2n)
   })
 })
+
+describe('deal-observer-backend built in actor event observer', () => {
+  let pgPool
+  const makeRpcRequest = async (method, params) => {
+    switch (method) {
+      case 'Filecoin.ChainHead':
+        return parse(JSON.stringify(chainHeadTestData))
+      case 'Filecoin.GetActorEventsRaw':
+        return parse(JSON.stringify(rawActorEventTestData)).filter(e => e.height >= params[0].fromHeight && e.height <= params[0].toHeight)
+      default:
+        console.error('Unknown method')
+    }
+  }
+  before(async () => {
+    pgPool = await createPgPool()
+    await migrateWithPgClient(pgPool)
+  })
+
+  after(async () => {
+    await pgPool.end()
+  })
+
+  beforeEach(async () => {
+    await pgPool.query('DELETE FROM active_deals')
+  })
+  it('deal observer loop function stores all retrievable active deals if database is empty', async () => {
```

Comment:
```suggestion
  it('stores all retrievable active deals if database is empty', async () => {
```

Reply:
In behavior driven tests, these calls should read like natural sentences (it stores...)